docs/add-ins/addin-tutorial.md

```diff
@@ -304,7 +304,15 @@ Since you previously installed the add-in from a file, you must reinstall it in
 
 1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to reinstall the add-in using the updated **manifest.xml** file.
 
-After you've completed these steps, you should see two new buttons in the ribbon of the message compose window: **Insert gist** and **Insert default gist**. 
+After you've reinstalled the add-in, you can verify that it installed successfully by checking for the options **Insert gist** and **Insert default gist** in the message compose window. Note that nothing will happen if you select either of these items, because you haven't yet specified the code that runs when these items are selected.
+
+- If you're running this add-in in Outlook 2016 or later for Windows, you should see two new buttons in the ribbon of the message compose window: **Insert gist** and **Insert default gist**. 
+
+    ![A screenshot of the ribbon in Outlook for Windows with the add-in's buttons highlighted](images/addin-tutorial/add-in-buttons-windows.png)
```

Comment:
I've noticed we sometimes end alt-text with a period and sometimes don't. I couldn't find anything in the style guide about this. Thoughts?

Reply:
In general, I'd say that if it's not a complete sentence (as most alt text descriptions aren't), then it shouldn't have a period.